### ISSUE TITLE
Calling `toTex` on node that contains call to `derivative()` causes exception

### ISSUE THREAD

**Issue by ovk** [josdejong/mathjs#1103]
Derivative function cannot be converted to TeX expression. When calling `toTex` on node that contains a call to `derivative` function the following exception is thrown
```
TypeError: Unexpected type of argument in function _derivTex (expected: SymbolNode, actual: ConstantNode, index: 1)
```
Here's a minimal example that reproduces this issue: https://jsfiddle.net/hhpxvprx

**Comment by joelhoover** [comment]
I've investigated this issue, and I believe it is fixed with #1104.

The issue here is that `math.parse('derivative("x", "x")')` will parse `derivative` as a function and create a `ConstantNode` with `valueType = 'string'` for both instances of `"x"`, and `derivative.toTex()` simply calls `_derivTex` with those `ConstantNode`s. However, `_derivTex` expects that it's arguments are the `Node`s that the strings represent (two `SymbolNode`s in this case), and so throws a `TypeError` because those types mismatch. This can be fixed by simply adding overloads to `_derivTex` that check if the arguments are `ConstantNode`s representing strings, and then parsing the string contents.

**Comment by josdejong** [comment]
This issue is fixed now by Joel, the fix will be available in the first next release, `v5.0.0` (expected to be released within a week)
